README.md

```diff
@@ -6,20 +6,26 @@
 
 First, download a prebuilt copy of TigerBeetle.
 
-On macOS/Linux:
-
 ```console
-git clone https://github.com/tigerbeetle/tigerbeetle && cd tigerbeetle && ./bootstrap.sh
+# macOS
+wget https://github.com/tigerbeetle/tigerbeetle/releases/latest/download/tigerbeetle-universal-macos.zip && unzip tigerbeetle-universal-macos.zip
```

Comment:
I think on Mac’s we have curl rather than wget? In general, looking at https://daniel.haxx.se/docs/curl-vs-wget.html curl seems a bit more appropriate

Reply:
Let’s also add `&& ./tigerbeetle —version`. [As I like to complain](https://matklad.github.io/2022/10/19/why-linux-troubleshooting-advice-sucks.html), it’s important to show not only what to do, but also some way to verify that it did work